### ISSUE TITLE
Funds sent to Sparrow generated address confirmed, but never arrived in Sparrow wallet v1.6.1

### ISSUE THREAD

**Issue by Salubrey** [sparrowwallet/sparrow#463]
I transfered SATs from the Strike app and they are not showing up in the Sparrow wallet. It was confirmed by Strike app and it is also confirmed on the blockchain however the amount is not showing up in the wallet.

I found a previous user had the same issue so I followed the steps recommended to no avail. That post is [here](https://github.com/sparrowwallet/sparrow/issues/125).

I checked the addresses receive log within the Sparrow wallet and the receive address was generated from the my wallet.

The transaction was in Sparrow wallet 1.6.0 I updated to 1.6.1 and the transaction has not gone through. The wallet is a BIP39 wallet. The receive address has never been used before. I have transfered SATs from Strike app before to fresh receive addresses before without a hitch. 

Any support  would be greatly appreciated.

**Comment by Transisto** [comment]
Did you see the transaction on a block explorer?

Did you try refreshing the wallet?

<img width="159" alt="image" src="https://user-images.githubusercontent.com/388716/158036425-7c0a2b25-6b65-4a94-8a86-df9021262f26.png">

**Comment by Salubrey** [comment]
Transisto thank you for the response.

Yes it does appear on a block explorer. I also refreshed the Sparrow wallet and the transaction in question did not appear.

It gets weirder. As a test, I copied the receive address (the one that the Sparrow wallet can not sync with) out of the "receive addresses log"  and sent it a few SATs, my hopes was maybe the Sparrow wallet would recognize it self. Those newly sent SATs where sent, receive,  and confirmed on a block explorer. The Sparrow wallet never received those new SATs. I can see them on the blockchain though. 

Since the initial missing transaction I have sent and received (2 other different transactions) however the one in question is still missing.

**Comment by craigraw** [comment]
> I checked the addresses receive log within the Sparrow wallet and the receive address was generated from the my wallet.

Just to be certain - do you mean you can see the address in the Addresses tab? 

What server are you connected to?

**Comment by Salubrey** [comment]
Thanks craigraw.

Yes that is correct from the addresses tab. I am running a full node locally, Bitcoin Core version v22.0.0.

I verified  the transactions using blockchain.com explorer.

**Comment by craigraw** [comment]
This is probably an indexing issue on Bitcoin Core. Try forcing a rescan by going to Settings tab > Advanced and changing the wallet birth date to an earlier date (a day or two previous to your first transaction). When you apply the change, Sparrow will ask Bitcoin Core to rescan from that date.

**Comment by Salubrey** [comment]
Craigraw it worked!

Thank you so much. After the rescan it found the transaction, however the amount of SATs went down, prior to the rescan I did backup the Sparrow wallet though... when I opened the back up everything was as it should be, all the SATs where there.

This experience has taught me much about the Sparrow wallet and the blockchain as a whole. Much appreciated. Thank you so much this has made my day.
